File-Manifests/NeMO-Audit-Manifests/README.md

```diff
@@ -0,0 +1,11 @@
+# NeMO Audit File Manifests
+
+Directory for File Manifests and Specimen Inventories for the data collections included in the NeMO file audit for 2021 Q1. 
+For each project, CSVs were generated for:
+
+1. A Specimen Inventory of samples in the BCDC inventory that have files at NeMO, with an added column of 'Files per Sample'
+2. A File Manifest generated for all matched files in the NeMO audit list, including associated Sample ID, NeMO ID, and File URI at NeMO, 
+as well as checksums provided by NeMO when available 
+4. A Specimen Inventory of samples in the BCDC inventory that had **no** matched files in the NeMO audit
+
```

Comment:
Add additional information such as:
- Note that there is one set of csvs per "project", and each csv may have multiple data collections included. The data collection association is provided in column Q of the Sample Inventory csvs, and column E of the file manifest.